cfme/containers/provider/openshift.py

```diff
@@ -30,7 +30,7 @@ def create(self, validate_credentials=True, **kwargs):
         # probably a js wait issue, not reproducible manually
         super(OpenshiftProvider, self).create(validate_credentials=validate_credentials, **kwargs)
 
-    def _href(self):
+    def href(self):
```

Comment:
LOL, forgot to say that I've added this method (https://github.com/ManageIQ/integration_tests/pull/3869/files) so there is no secondary intent AFAIK :)